model/core/arbiter.cc

```diff
@@ -54,6 +66,12 @@ Arbiter::Arbiter(Ptr<Node> this_node, NodeContainer nodes) {
 }
 
 uint32_t Arbiter::ResolveNodeIdFromIp(uint32_t ip) {
+    if(m_tap_bridge_enable){
+        if ((ip & 0xFF) != 1) {
+            ip = (ip & 0xFFFFFF00) | 2; 
```

Comment:
Passing in a parameter ip that is empty will result in an error. Need to add security check

Reply:
why？ 0 & 0xFF = 0 so the code ”ip = (ip & 0xFFFFFF00) | 2“ will not run